### DISCUSSION TITLE
How to enable the new hunter samples (Forest Keeper, Manticoil and Tulip Snake)

### DISCUSSION THREAD

**Discussion by McGodMan** [Malcolm-Q/LC-LateGameUpgrades#454] [Q&A] [answered]
I've been messing around with the config to try and enable the new samples but nothing I try seems to be working, how do I enable them?

I've already tried several different names for the entities in the config (specifically the samples dropping at each tier section of hunter) but nothing gets them to spawn when the entities are killed.

Many thanks!

**Comment by chibranche** [comment, marked as answer]
I took a look at the code, there doesn't seems to be one for the buttler yet, but here are the aliases I found for the Tulip Snake: 

tulip snake, FlowerSnake, Flower Snake, Snake

I didn't test myself but if this is implemented, any of these values should work

**Reply by McGodMan** [reply]
Just loaded the game up today and they seem to be spawning now, no idea why but thanks for the answer anyway, I had tulip snake in the config.
